lib/2_uppercases_all_letters.js

```diff
@@ -6,6 +6,7 @@
 
 module.exports = {
   uppercases_all_letters: function(str){
-    // Your code goes here.
+    string_upp = str.toUpperCase();
+      return string_upp
```

Comment:
Just to be consistent 😄  Line 10 can be brought back one indentation to be in line with Line 9.